hetznerbot/helper/hetzner.py

```diff
@@ -243,7 +253,30 @@ def check_offer_for_subscriber(session, subscriber):
     session.commit()
 
 
-def format_offers(subscriber, offer_subscriber, get_all=False):
+async def notify_about_new_cpu(session, bot):
+    """Check for any cpu for which no stats are present. If so, let the admin know."""
+    query = (
+        select(Offer.cpu)
+        .distinct()
+        .filter(Offer.cpu.notin_(session.query(Cpu.cpu_name)))
+    )
+
+    new_cpus = session.execute(query).all()
+    if len(new_cpus) == 0:
+        return
+
+    info = "Please add info about these cpus:"
+    for cpu in new_cpus:
+        info += f"{cpu}\n"
```

Comment:
Could you add all new cpu names to a list in [context.bot_data](https://docs.python-telegram-bot.org/en/stable/telegram.ext.callbackcontext.html#telegram.ext.CallbackContext.bot_data)?

That way we can check whether the admin has already been notified about a certain since the bot has started. Otherwise any admin will get a message about a new CPU every 2 minutes :D